src/lib/components/Footer/Footer.svelte

```diff
@@ -0,0 +1,145 @@
 35.8864 46.3819C35.7709 46.3819 35.6583 46.3793 35.5431 46.3767C35.4892 46.3742 35.4357 46.3716 35.3818 46.369C35.2971 46.3665 35.2127 46.3613 35.1309 46.3536C35.059 46.3511 34.9901 46.3434 34.9208 46.3357C34.8185 46.328 34.7158 46.3177 34.6135 46.3049C34.4829 46.2895 34.3549 46.2715 34.2244 46.251C34.0964 46.2304 33.971 46.2073 33.843 46.1817C33.7942 46.1714 33.7432 46.1637 33.6944 46.1509C33.4766 46.1047 33.2617 46.0537 33.0464 45.9921C32.9951 45.9793 32.9441 45.9639 32.8928 45.9485C32.6265 45.8689 32.3599 45.7794 32.0962 45.6771C31.9939 45.6386 31.8912 45.5979 31.7889 45.5543C31.5227 45.4417 31.2586 45.3163 30.9975 45.1803C30.9234 45.1418 30.8464 45.1007 30.7723 45.0575C30.6953 45.0164 30.6212 44.9753 30.5468 44.9295C30.4723 44.8859 30.3983 44.84 30.3242 44.7938C30.2549 44.7502 30.1859 44.7069 30.1166 44.6607C30.1115 44.6582 30.1089 44.6556 30.1038 44.653C30.0117 44.5914 29.9193 44.5276 29.8273 44.4635C29.707 44.3788 29.5867 44.2919 29.4686 44.1998L35.8864 37.7832H35.8868ZM35.8868 52.3116C31.9481 52.3116 28.2192 50.9366 25.2538 48.4121L27.5535 46.1127C27.7817 46.2972 28.012 46.4739 28.2474 46.6404C28.327 46.6968 28.4036 46.7504 28.4828 46.8043C28.565 46.8607 28.6442 46.9117 28.7238 46.963C28.8059 47.0169 28.8851 47.0679 28.9669 47.1167C29.2922 47.3165 29.6226 47.5031 29.9604 47.6696C30.0271 47.703 30.091 47.7363 30.1577 47.7668C30.2729 47.8232 30.3909 47.8768 30.5087 47.9281C30.5754 47.9589 30.6418 47.9871 30.7085 48.015C30.7778 48.0458 30.8493 48.0766 30.9187 48.1019C31.0004 48.1353 31.0826 48.1686 31.1644 48.1991C31.1952 48.2119 31.2234 48.2222 31.2539 48.2324C31.428 48.2992 31.6048 48.3604 31.7841 48.4169C32.1681 48.5448 32.5576 48.6501 32.9492 48.7395C33.0109 48.7524 33.0695 48.7652 33.1311 48.778C33.5973 48.8778 34.0685 48.9522 34.5398 48.998C34.6168 49.0057 34.6934 49.0134 34.7675 49.0211C34.9571 49.0365 35.1441 49.0494 35.3308 49.0545C35.4254 49.0596 35.5178 49.0622 35.6099 49.0648C35.7943 49.0699 35.9788 49.0699 36.1629 49.0648C36.255 49.0622 36.3474 49.0596 36.442 49.0545C36.629 49.0494 36.816 49.0365 37.0053 49.0211C37.0794 49.0134 37.1538 49.0057 37.2305 49.0006C37.753 48.9467 38.2727 48.8624 38.7876 48.7472C38.8668 48.7293 38.949 48.7113 39.0282 48.6908C39.5632 48.5602 40.091 48.3963 40.6084 48.1991C40.6906 48.1683 40.7723 48.1349 40.8541 48.1019C40.9234 48.0762 40.9949 48.0454 41.0643 48.015C41.131 47.9868 41.1974 47.9585 41.2641 47.9281C42.1094 47.5541 42.9236 47.0855 43.6893 46.5223C43.7663 46.4658 43.8404 46.4097 43.9148 46.3533C44.0172 46.2763 44.1173 46.1971 44.217 46.1153H44.2196L44.2684 46.1615L46.5194 48.4121C43.551 50.937 39.8251 52.3116 35.8864 52.3116H35.8868ZM48.4201 46.5095L46.1691 44.2614V44.2589L46.1204 44.2127V44.2101C48.0409 41.862 49.0835 38.9506 49.0835 35.8805C49.0835 32.3572 47.7109 29.044 45.2189 26.55C40.3942 21.7283 32.7318 21.4291 27.5535 25.6487L25.2538 23.3493C28.2195 20.8248 31.9481 19.4498 35.8868 19.4498C40.2761 19.4498 44.4044 21.1578 47.5084 24.2612C53.5958 30.3501 53.9005 40.0624 48.4201 46.5095Z"
```

Comment:
Put this icons in `src/lib/assets/icons` and import it you might have to download `vite-plugin-svelte-svg`  as well in order to import svg

```
npm install -D vite-plugin-svelte-svg
```

```typescript
// svelte.config
import { vitePreprocess } from '@sveltejs/kit/vite';
import svg from 'vite-plugin-svelte-svg';

export default {
  preprocess: vitePreprocess(),
  kit: {
    alias: {
      $lib: 'src/lib',
    },
    vite: {
      plugins: [svg()],
    },
  },
};
```

Reply:
version vite and svelte svg unmatch,  so just import svg in <img>